index.js

```diff
@@ -6,19 +6,31 @@ var fs        = require('fs')
   , envWriter = require('env-writer')()
   , envData
 
-module.exports = function (envFile) {
+module.exports = function (envFile, callback) {
+  if (typeof envFile === 'function') {
+    callback = envFile;
+    envFile = undefined;
+  }
+
+  if (!callback) {
+    callback = function() { /* noop */ }
+  }
 
   if (typeof envFile === 'undefined')
-    envFile = '.env'
+    envFile = '.env';
 
   try {
-    envData = fs.readFileSync(envFile)
+    envData = fs.readFileSync(envFile);
```

Comment:
Sure,
because readFileSync is blocking and we are adding a way to asynchronously load our env, it might be nice to make the whole process non blocking and asynchronous like... 
```
fs.createReadStream(envFile)
  .pipe(envReader)
  .pipe(envParser)
  .pipe(envWriter)
  .on('end', callback)
```
but only when the callback is provided. 
It should remain blocking if a callback is not provided by continuing to load the env file using readFileSync

Does that make sense?